tests/components/sighthound/test_image_processing.py

```diff
@@ -91,3 +95,23 @@ def capture_person_event(event):
     state = hass.states.get(VALID_ENTITY_ID)
     assert state.state == "2"
     assert len(person_events) == 2
+
+
+async def test_save_image(hass, mock_image, mock_detections):
+    """Save a processed image."""
+    VALID_CONFIG_SAVE_FILE = deepcopy(VALID_CONFIG)
+    VALID_CONFIG_SAVE_FILE[ip.DOMAIN].update({sh.CONF_SAVE_FILE_FOLDER: TEST_DIR})
+    await async_setup_component(hass, ip.DOMAIN, VALID_CONFIG)
```

Comment:
We need to pass the copied and updated config when setting up the component and platform.

Reply:
OMG how blind am I :-)